lib/job-simulation.rb

```diff
@@ -2,13 +2,43 @@
 require './Queue.rb'
 
 class JobSimulation
-  attr_reader :workers, :waiting, :roll
+  attr_reader :workers, :waiting, :roll,:count
 
-  def initialize (jobs_available, job_seekers)
+  def initialize(jobs_available, total_jobs)
+    #setting total numbers
+    @count = total_jobs
+    all = Array.new
+    (0..count).each{|number|all.push("Worker ##{number+1}")}
+    #splitting workers between those working and those waiting
+    starting_workers = all.slice(0..jobs_available)
+    waiting_workers = all.slice(jobs_available + 1..-1)
```

Comment:
This is creative~ but it uses auxiliary storage that I didn't say you could create (I guess I didn't say you couldn't either ;) ). Could you use loops to create this without needing the use of an array?